components/DuplicateWarningPanel/DuplicateWarningPanel.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
+import cx from 'classnames';
+import s from './DuplicateWarningPanel.module.scss';
+import { LegacyResident } from 'types';
+import axios from 'axios';
+import ResidentsTable from 'components/Search/results/ResidentsTable';
+
+interface Props {
+  newResident: Record<string, unknown>;
+}
+
+const DuplicateWarningPanel = ({
+  newResident,
+}: Props): React.ReactElement | null => {
+  const [matchingResidents, setMatchingResidents] = useState<LegacyResident[]>(
+    []
+  );
+
+  useEffect(() => {
+    axios
+      .get(`/api/residents`, {
+        headers: { 'x-api-key': process.env.AWS_KEY },
+        params: {
+          first_name: newResident.firstName,
+          last_name: newResident.lastName,
+        },
+      })
+      .then(({ data }) => setMatchingResidents(data.residents));
```

Comment:
What happens if this errors? We aren't handling the `.catch` right now (also a good argument for using `useResidents()`).

Reply:
since it's not the main purpose of the page, failing silently is probably the correct (and current) behaviour. it would be more confusing to render an error message.